choreography/src/ingestion/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM openjdk:8-jre-alpine as base
+FROM mcr.microsoft.com/java/jre:8-zulu-debian10 as base
```

Comment:
IIRC back then the reason why `alpine` has been chosen is because it was the lightweight. Just curious, any particular recommendation to move over `debian10`?

Reply:
It's all that MCR hosts. And if we pull from a public repo, we have to document a workaround. The build took just about as along.

Reply:
I'm seeing a zulu on alpine also. Am I missing something?

```
REPOSITORY                   TAG                 IMAGE ID            CREATED             SIZE
mcr.microsoft.com/java/jre   8-zulu-debian10     8a0dee38b595        3 days ago          247MB
mcr.microsoft.com/java/jre   8-zulu-alpine       d3fc782975fb        3 days ago          144MB
```

Reply:
Did you see both jre and jdk? It's possible I missed them. But I thought I only saw one.

Reply:
ah, `maven-zule-alpine` doesn't seem to be available. Yeah, I agree mixing them up without proper testing in place is not a good idea.